src/benchmark.cc

```diff
@@ -115,7 +115,13 @@ std::string* GetReportLabel() {
 
 // Global variable so that a benchmark can report an error as a human readable
 // string. If error_message is null no error occurred.
-static std::atomic<const char*> error_message = ATOMIC_VAR_INIT(nullptr);
+#if defined(BENCHMARK_WORKAROUND_VS_2013)
```

Comment:
it should do.

`#if _MSC_VER <= 1800` alone won't as `_MSC_VER` will be 0 if undefined.